External/FEXCore/Source/Interface/Core/JIT/Arm64/JIT.cpp

```diff
@@ -1987,7 +2011,41 @@ void *JITCore::CompileCode([[maybe_unused]] FEXCore::IR::IRListView<true> const
         break;
         }
         case 8: {
-          udiv(GetDst<RA_64>(Node), GetSrc<RA_64>(Op->Header.Args[0].ID()), GetSrc<RA_64>(Op->Header.Args[2].ID()));
+          uint64_t SPOffset = AlignUp((RA64.size() + 1) * 8, 16);
+
+          sub(sp, sp, SPOffset);
+          int i = 0;
+          for (auto RA : RA64) {
+            str(RA, MemOperand(sp, i * 8));
+            i++;
+          }
+          str(lr,       MemOperand(sp, RA64.size() * 8 + 0 * 8));
+
+          mov(x0, GetSrc<RA_64>(Op->Header.Args[1].ID()));
+          mov(x1, GetSrc<RA_64>(Op->Header.Args[0].ID()));
+          mov(x2, GetSrc<RA_64>(Op->Header.Args[2].ID()));
+
+#if _M_X86_64
```

Comment:
Why is this ifdef needed?

Reply:
This ifdef is needed for running the vixl simulator for the AArch64 JIT on an x86 host.